src/coreclr/src/System.Private.CoreLib/src/System/RuntimeType.CoreCLR.cs

```diff
@@ -2325,7 +2325,6 @@ private static bool FilterApplyPrefixLookup(MemberInfo memberInfo, string name,
         internal static readonly RuntimeType ValueType = (RuntimeType)typeof(System.ValueType);
 
         private static readonly RuntimeType ObjectType = (RuntimeType)typeof(object);
```

Comment:
`ObjectType ` doesn't seem to be used anywhere now, can it be removed like `StringType` was?

Reply:
It is used in `RuntimeType.cs`, the other part of this partial class. Removing that one would be a tiny deoptimization.